### DISCUSSION TITLE
Thanks for NAPS

### DISCUSSION THREAD

**Discussion by Have-Purple** [cyanfish/naps2#33] [General]
Thanks for NAPS.  Great program that I use weekly!

**Comment by gomiunik** [comment]
By far the best free scan-to-PDF + OCR software.

**Comment by diskostu** [comment]
I just installed the update 7.3.0. The settings dialog is so useful - "keep images between sessions", yay!

Thanks, devs. You are doing an awesome job with this app.

**Comment by klausijoe** [comment]
Thank you so much for  dark mode! My eyes are celebrating!

**Comment by MaFserv** [comment]
Amazing scanning tool loaded with features. 
Thanks to all those who have put their efforts in developing this special tool.

**Comment by toxic0berliner** [comment]
Yeah, I just wanted to do exactly the same and just give many many thanks as this is an awesome software, congrats to the devs and again many thanks for sharing it as FOSS !

**Comment by RNavarro3000** [comment]
One of the very few opensource software's I've used and then immediately donated after modifying one file.  So intuitive to use, packed full of features, and also supports Linux!

**Comment by ojessie** [comment]
Thanks for the application. Highly appreciated. As a mover from MS to Ubuntu with limited knowledge, it was easy to install and use it.
